src/index.html

```diff
@@ -4,13 +4,53 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bar">
+      <form
+        aktion="https://server.api"
+        method="get"
+        data-qa="big"
+        class="big-form">
+        <label for="big_input" class="label">
```

Comment:
When you wrap up input with label no need to specify `for`

Reply:
Thanks